modules/grizzly/src/main/java/org/glassfish/grizzly/ssl/SSLEngineConfigurator.java

```diff
@@ -272,6 +273,21 @@ public SSLEngineConfigurator setWantClientAuth(boolean wantClientAuth) {
         return this;
     }
 
+    /**
+     * Apply {@link SSLParameters} to this SSLEngineConfigurator.
```

Comment:
> Apply {@link SSLParameters} to this SSLEngineConfigurator.

Might want to document that only the cipherSuites, protocols, needClientAuth and wantClientAuth properties are applied?